app/src/main/java/org/kiwix/kiwixmobile/main/AddNoteDialog.java

```diff
@@ -70,36 +70,51 @@
 
   private Unbinder unbinder;
 
+  private String zimFileName;
   private String zimFileTitle;
   private String articleTitle;
-  private String zimNoteDirectoryName;  // Corresponds to "ZimFileName" of "{External Storage}/Kiwix/Notes/ZimFileName/ArticleUrl.txt"
-  private String articleNotefileName;   // Corresponds to "ArticleUrl" of "{External Storage}/Kiwix/Notes/ZimFileName/ArticleUrl.txt"
+  // Corresponds to "ZimFileName" of "{External Storage}/Kiwix/Notes/ZimFileName/ArticleUrl.txt"
+  private String zimNoteDirectoryName;
+  // Corresponds to "ArticleUrl" of "{External Storage}/Kiwix/Notes/ZimFileName/ArticleUrl.txt"
+  private String articleNotefileName;
   private boolean noteFileExists = false;
-  private boolean noteEdited = false;   // Keeps track of state of the note (whether edited since last save)
```

Comment:
why has the visibility of so many things changed to package instead of private?

Reply:
They were being accessed from clickListeners of confirmation-dialog, so keeping them private caused a synthetic accessor warning
Should I revert?

Reply:
hmmm I am considering ignoring SyntheticAccessor because with the addition of R8 our method count in release is under 30K. Continue as you are and I will start a discussion elsewhere